src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write code here
+function toNumber(numStr) {
```

Comment:
```suggestion
function convertToNumber(numStr) {
```
Would be great to follow these rules on naming functions:
![image](https://user-images.githubusercontent.com/96409563/211902133-3cdc8005-f1d3-4056-9d08-3b327a480954.png)